src/AzureAuth.Test/CommandMainTest.cs

```diff
@@ -342,12 +343,46 @@ public void TestEvaluateOptionsWithoutAliasMissingRequiredOptions()
             subject.EvaluateOptions().Should().BeFalse();
             this.logTarget.Logs.Should().Contain(new[]
             {
-                "The --resource field is required.",
+                "The --resource field or the --scope field is required.",
                 "The --client field is required.",
                 "The --tenant field is required.",
             });
         }
 
+        /// <summary>
+        /// The test to evaluate options without resource but with scopes.
+        /// </summary>
+        [Test]
+        public void TestEvaluateOptionsWithOverridedScopes()

```

Comment:
Let's add tests for the 2 other combos as well:
1. Neither resource or scope is given
2. Resource is given but no scopes

Reply:
Did those tests not already exist?

Reply:
Those tests are already covered.